src/components/Instruction.tsx

```diff
@@ -20,14 +19,18 @@ export function Instruction({
       <Box style={styles.bullet} backgroundColor={`${colorMode}.black`}></Box>
       <Text color={`${colorMode}.secondaryText`} style={styles.infoText}>
         {typeof text === 'string'
-          ? text.split(/\b(https?:\/\/[^\s]+)\b/).map((part) => {
+          ? text.split(/\b(https?:\/\/[^\s]+)\b/).map((part, index) => {
               if (part.match(/^https?:\/\//)) {
                 return (
-                  <TouchableOpacity onPress={() => openLink(part)} style={styles.linkContainer}>
-                    <Text color={`${colorMode}.greenWhiteText`} style={styles.linkText} bold>
-                      {part.replace('https://', '').replace('http://', '')}
-                    </Text>
-                  </TouchableOpacity>
+                  <Text
+                    key={index}
```

Comment:
the key was giving me some errors in certain screens, is it necessary?

Reply:
we can remove it.